Makefile

```diff
@@ -55,3 +64,30 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+.PHONY: bench-deps
+bench-deps:
+	@echo "Installing benchstat tool"
+	rm -rf ${BENCH_PATH}
+	mkdir ${BENCH_PATH}
+	go clean -testcache
+	cd ${BENCH_PATH} && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	rm -rf ${BENCH_PATH}/perf
+
+.PHONY: bench
+${BENCH_FILE} bench:
+	@echo "Running benchmark tests from the current branch"
+	go test ${TEST_PATH} -bench=. -run=^Benchmark -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 \
+		| tee ${BENCH_FILE}
+	benchstat ${BENCH_FILE}
+
+${REFERENCE_FILE}:
+	@echo "Running benchmark tests from master for using results in bench-diff target"
+	cd ${BENCH_PATH} && rm -rf go-tarantool && git clone https://github.com/tarantool/go-tarantool.git
+	cd ${BENCH_PATH}/go-tarantool && go test ${BENCH_PATH}/go-tarantool/... -bench=. -run=^Benchmark -benchmem \
+ 		-benchtime=${DURATION} -count=${COUNT} 2>&1 | tee ${REFERENCE_FILE}
+	rm -rf ${BENCH_PATH}/go-tarantool
+
+bench-diff: ${BENCH_FILES}
+	@echo "Comparing performance between master and the current branch"
+	benchstat ${BENCH_FILES}
```

Comment:
Nit: I propose to describe what does mean 'old' and 'new' and suppress command-line output for benchstat.

```diff
 bench-diff: ${BENCH_FILES}
        @echo "Comparing performance between master and the current branch"
-       benchstat ${BENCH_FILES}
+       @echo "'old' is a version in master branch, 'new' is a version in a current branch"
+       @benchstat ${BENCH_FILES}
```

Reply:
Feel free to ignore.

Reply:
Fixed.